src/amazonica/aws/ecs.clj

```diff
@@ -0,0 +1,8 @@
+(ns amazonica.aws.ecs
+  (:require [amazonica.core :as amz])
+  (:import [com.amazonaws.services.ecs AmazonECS AmazonECSClient]))
+
+(amz/set-client AmazonECS *ns*)
```

Comment:
I initially set the client to the interface before I had generated listing of methods for the test.  When I fixed this to reference the client I noticed that some methods were missing and realized that set-client was not walking the inheritance hierarchy and the client was missing set-region and shutdown.  Although it occurs to me now that even though a method is generated for the interface it probably wouldn't work since it probably wouldn't use the same client instance and fail to create instance of the interface when called. 

I have needed in previous releases to set-endpoint manually on elasticmapreduce when using credentials form sts-assume-role (really hacky workaround of rebinding #'*credientials* to new credentials object that includes session token) so I figured it might be needed to call set-region or set-endpoint in some cases.  (Which I just discovered is now excluded in the latest release).  The is probably 2 separate issues (support STS assume role) and possibly expose methods from parent classes.